packages/@ourworldindata/grapher/src/core/GrapherState.tsx

```diff
@@ -291,10 +293,10 @@ export class GrapherState {
     }
 
     /**
-     * Used to highlight an entity at a particular time in a line chart.
+     * Used to highlight particular times in a line chart.
      * The sparkline in map tooltips makes use of this.
      */
-    entityYearHighlight: EntityYearHighlight | undefined = undefined
+    highlightedTimesInLineChart?: Time[]
```

Comment:
This is extended so that multiple times can be highlighted in a line chart (we need this for the map tooltips when faceted)